devel/models.py

```diff
@@ -40,6 +40,7 @@ class UserProfile(models.Model):
     website = models.URLField(max_length=200, null=True, blank=True)
     website_rss = models.URLField(max_length=200, null=True, blank=True,
                                   help_text='RSS Feed of your website for planet.archlinux.org')
+    social = models.URLField(max_length=200, null=True, blank=True)
```

Comment:
Maybe add some `help_text` like above, otherwise it may not be clear that it expects a URL when the user sees "Social:" in the web form

Reply:
Right! :man_facepalming: